MultipleImages/story.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { storiesOf } from '@kadira/storybook';
+import MultipleImages from './index';
+
+storiesOf('MultipleImages')
+  .add('Default', () => (
+    <MultipleImages
+      urls={[
+        'http://lorempixel.com/400/400/cats/',
+        'http://lorempixel.com/400/400/cats/',
+        'http://lorempixel.com/400/400/cats/',
+        'http://lorempixel.com/400/400/cats/',
+      ]}
+      width={'20em'}
+      height={'20em'}
+    />
```

Comment:
Woot! Looks awesome, @stevemdixon! 😄  I'd love to see some extra tests here - maybe we could have some urls that are for tall images and some with wider images? Loving this! Awesome stuff, @stevemdixon! 🙌